NeoBurgerGovernanceToken.cs

```diff
@@ -0,0 +1,84 @@
+        public override byte Decimals() => 8;
+        public override string Symbol() => "NOBUG";
+
+        public static UInt160 TEE() => (UInt160)Storage.Get(Storage.CurrentContext, new byte[] { PREFIX_TEE });
+        public static Iterator PassedProposals() => Storage.Find(Storage.CurrentContext, new byte[] { PREFIX_PASSED_PROPOSAL }, FindOptions.RemovePrefix);
+        public static Iterator PausedProposals() => Storage.Find(Storage.CurrentContext, new byte[] { PREFIX_PAUSED }, FindOptions.RemovePrefix);
+
+        public static void _deploy(object data, bool update)
+        {
+            Storage.Put(Storage.CurrentContext, new byte[] { PREFIX_TEE }, TEE_ADDRESS);
+        }
+
+
+        public static void ProposalPassed(BigInteger proposal_id)
+        {
+            ExecutionEngine.Assert(Runtime.CheckWitness((UInt160)Storage.Get(Storage.CurrentContext, new byte[] { PREFIX_TEE })));
+            Storage.Put(Storage.CurrentContext, (ByteString)new byte[] { PREFIX_PASSED_PROPOSAL } + (ByteString)proposal_id, Runtime.Time);
+        }
+
+        public static void ExecutionPaused(BigInteger proposal_id)
+        {
+            ExecutionEngine.Assert(Runtime.CheckWitness((UInt160)Storage.Get(Storage.CurrentContext, new byte[] { PREFIX_TEE })));
+            Storage.Put(Storage.CurrentContext, (ByteString)new byte[] { PREFIX_PAUSED } + (ByteString)proposal_id, 1);
+        }
+
+        public static void ExecutionResumed(BigInteger proposal_id)
+        {
+            ExecutionEngine.Assert(Runtime.CheckWitness((UInt160)Storage.Get(Storage.CurrentContext, new byte[] { PREFIX_TEE })));
+            Storage.Delete(Storage.CurrentContext, (ByteString)new byte[] { PREFIX_PAUSED } + (ByteString)proposal_id);
+        }
+
+        public static object ExecuteProposal(BigInteger proposal_id, UInt160 scripthash, string method, ByteString[] args)
+        {
+            ExecutionEngine.Assert(Runtime.CheckWitness((UInt160)Storage.Get(Storage.CurrentContext, new byte[] { PREFIX_TEE })));
+            ByteString proposalPassedKey = (ByteString)new byte[] { PREFIX_PASSED_PROPOSAL } + (ByteString)proposal_id;
+            if (Runtime.Time - (BigInteger)Storage.Get(Storage.CurrentContext, proposalPassedKey) < PUBLICITY_PERIOD_BEFORE_EXECUTION)
+                throw new System.Exception("not enough publicity period");
+            if ((BigInteger)Storage.Get(Storage.CurrentContext, (ByteString)new byte[] { PREFIX_PAUSED } + (ByteString)proposal_id) > 0)
+                throw new System.Exception("proposal paused");
+            Storage.Delete(Storage.CurrentContext, proposalPassedKey);
+            return Contract.Call(scripthash, method, CallFlags.All, args);
+        }
+
+        public static void ChangeTEE(UInt160 newTEE)
+        {
+            ByteString newTEEBytearray = (ByteString)new byte[] { PREFIX_TEE };
+            ExecutionEngine.Assert(Runtime.CheckWitness((UInt160)Storage.Get(Storage.CurrentContext, newTEEBytearray)));
```

Comment:
use `ExecutionEngine.Assert(Runtime.CheckWitness(Runtime.ExecutingScriptHash));` instead to ensure the timelock